### PR TITLE
Remove checking for Read permission

### PR DESCRIPTION
**Pull request by boy-uber**

### CODE REVIEW

File: services/frontendhost/frontend.go [hunk 1]
```diff
@@ -666,7 +666,7 @@ func (h *Frontend) CreateDestination(ctx thrift.Context, createRequest *c.Create
 		dstResource := common.GetResourceURNOperateDestination(h.SCommon, createRequest.Path)
 		h.addPermissions(subject,
 			dstResource,
-			[]common.Operation{common.OperationRead, common.OperationUpdate, common.OperationDelete},
+			[]common.Operation{common.OperationUpdate, common.OperationDelete},
```
Comment (datoug) [inline services/frontendhost/frontend.go]: update the comment above.

Comment (datoug) [inline services/frontendhost/frontend.go]: can we remove the common.OperationRead enum?

File: services/frontendhost/frontend.go [hunk 2]
```diff
@@ -1002,7 +1002,7 @@ func (h *Frontend) UpdateDestination(ctx thrift.Context, updateRequest *c.Update
 		dstResource := common.GetResourceURNOperateDestination(h.SCommon, updateRequest.Path)
 		h.addPermissions(ownerSubject,
 			dstResource,
-			[]common.Operation{common.OperationRead, common.OperationUpdate, common.OperationDelete},
+			[]common.Operation{common.OperationUpdate, common.OperationDelete},
```
Comment (datoug) [inline services/frontendhost/frontend.go]: update comment above

File: services/frontendhost/frontend.go (hunk 1 above)
Comment (boy-uber) [inline services/frontendhost/frontend.go]: Yes

Comment (boy-uber) [inline services/frontendhost/frontend.go]: Good suggestion

**Comment by boy-uber** [comment]
Updated code per comments

**Pull request merged**